tutorials/Tutorial12_LFQA.ipynb

```diff
@@ -8,7 +8,9 @@
    "source": [
     "# Long-Form Question Answering\n",
     "\n",
-    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/deepset-ai/haystack/blob/master/tutorials/Tutorial12_LFQA.ipynb)"
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/deepset-ai/haystack/blob/master/tutorials/Tutorial12_LFQA.ipynb)\n",
+    "\n",
+    "This tutorial shows how to build and use a pipeline for generative question answering, in particular, a task usually referred to as Long-Form Question Answering (LFQA). These systems function by querying large document stores for relevant information and subsequently using the retrieved documents to generate accurate, multi-sentence answers. The retrieved documents related to a given query, colloquially called context passages, are not used merely as source tokens for extracted answers, but instead provide a larger context for the synthesis of original, abstractive long-form answers."
```

Comment:
How about changing it a bit to make the sentences simpler:
Follow this tutorial to learn how to build and use a pipeline for Long-Form Question Answering (LFQA). LFQA is a variety of the generative question answering task. LFQA systems query large document stores for relevant information and then use this information to generate accurate, multi-sentence answers. In a regular question answering system, the retrieved documents related to the query (context passages) act as source tokens for extracted answers. In an LFQS system, context passages provide the context the system uses to generate original, abstractive, long-form answers.

Reply:
Deal @agnieszka-m